tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -94,7 +94,7 @@ Nevertheless, there are quite a few pitfalls you can fall into when using the pl
 - Finally, you need to define a class such as `MyBusinessPartnerType` with specific annotations that represents the properties and their types of the result. For this you again need to know a lot of details about the OData service.
 
 
-### Virtual Data Model: The new way to OData
+### Virtual Data Model: The simpler way to OData
 
 
 Now that you have seen some of the possible pitfalls of the current approach, let's take a look at how the OData VDM of the SAP Cloud SDK simplifies the same task, as the SDK is able to incorporate more knowledge about the system that is being called.
```

Comment:
Don't use "we" or "us" unless you are referring to SAP.
-> remove "let's" (here and anywhere else in this tutorial)